src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -61,9 +62,28 @@ def wait_process(proc, wait_time):
         return procrtn
 
     @staticmethod
-    def cleanup_processes():
+    def cleanup_processes(self):
         """ Clean up cart processes, in case avocado/apricot does not. """
-        stop_processes(["localhost"], "'(crt_launch|orterun)'")
+        error_list = []
+        localhost = socket.gethostname().split(".")[0:1]
+        processes = "'\<(crt_launch|orterun)\>'"
```

Comment:
(lint) '\<'. String constant might be missing an r prefix.
(lint) '\>'. String constant might be missing an r prefix.